docs/05_operations/02_commands/04_Genx_CLI.md

```diff
@@ -0,0 +1,448 @@
+```
+
+### --ref
+
+Use the `--ref` parameter to specify the **branch**, **tag** or **commit** from a remote location, in case you do not want to use the main one.
+
+Here is an example of creating a new project named **myApp** based on positions-app-seed on the **develop** branch:
+
+```terminal
+npx -y @genesislcap/genx@latest init myApp -s positions-app-seed --ref develop
+```
+
+### --skip-optional-prompts
+
+Use the `-x` or `--skip-optional-prompts` parameter to skip all the optional questions when creating a new project. 
+
+Here is an example of creating a new project named **myApp** skipping all questions.
+
+```terminal
+npx -y @genesislcap/genx@latest init myApp -x
+```
+
+or
+
+```terminal
+npx -y @genesislcap/genx@latest init myApp --skip-optional-prompts
+```
+
+:::caution
+Creating a project using `-x` will create only sing the default configurations.
+:::
+
+### --insecure
+
+Use the `--insecure` parameter to skip all the SSL certification validation when creating a new project.
+
+Here is an example of creating a new project named **myApp** skipping the SSL certification validation.
+
+```terminal
+npx -y @genesislcap/genx@latest init myApp --insecure
+```
+
+### --remote
+
+Use the `--remote` parameter to use a remote-only seed. It will ignore any local seed.
+
+Here is an example of creating a new project named **myApp** with a seed called blank-app-seed (It will ignore any blank-app-seed locally).
+
+```terminal
+npx -y @genesislcap/genx@latest init myApp -s blank-app-seed --remote
+```
+
+### --log-level
+
+Use the `log-level` to choose between **info** (default) or **verbose** log level.
+
+Here is an example of creating a new project named **myApp** changing the log level to verbose:
+
+```terminal
+npx -y @genesislcap/genx@latest init myApp --log-level verbose
+```
+
+## Analyze
+
+The `analyze` command ...
+
+This command can be used with the following parameters:
+
+|Parameter | Argument  | Description |
+|----------|-----------|-------------|
+|`-b` or `--builder`|`<builder>` | Override default builder |
+| `-n` or `--no-open`| Don't launch browser window (default: true) |
+|`-e` or `--env` | `<ENV_VAR1=VAL1,ENV_VAR2=VAL2,ENV3>` | Set environment variables|
+|`-h` or `--help` | | |
+
+### --builder
+
+Use the `-b` or `--builder` when you want to override the default builder.
+
+Here is an example of... <!--Needs to provide some example-->
+
+### --no-open
+
+Use the `-n` or `--no-open` when you don't want to launch a browser window.
+
+Here is an example of... <!--Needs to provide some example-->
+
+### --env
+
+Use `e` or `--env` if you need to set environment variables.
+
+Here is an example of... <!--Needs to provide some example-->
+
+## Clean
+
+The `clean` command will delete the provided path. The given path is relative to the directory the command is ran.
+
+Here is an example to clean the project called **myApp**:
+
+```terminal
+npx -y @genesislcap/genx@latest clean myApp
+```
```

Comment:
In this case it'd be simpler to just delete that folder using native OS delete command.

It's more useful when you use it in a project:
`"clean": "genx clean",`

Running `npm run clean` will clear the dist folder (containing previously built artifact) and temporary compilation files.